docs/source/conf.py

```diff
@@ -1,43 +1,14 @@
-# Configuration file for the Sphinx documentation builder.
-
 import os
 import sys
+from sphinx_pyproject import SphinxConfig
```

Comment:
Using this package to load all the config vars from `pyproject.toml`. It's a little hacky. In the future I think we should try moving to Quarto for the docs site.